utils/dataloaders.py

```diff
@@ -677,6 +677,7 @@ def __len__(self):
     #     return self
 
     def __getitem__(self, index):
+        im0 = None
```

Comment:
Do you know why this is added?

Reply:
I will remove it, thanks, I added it to fix the ci tests